sup3r/utilities/log_interpolator.py

```diff
@@ -0,0 +1,679 @@
+"""Rescale ERA5 wind components according to log profile"""
+
+import logging
+import os
+from concurrent.futures import (
+    ProcessPoolExecutor,
+    ThreadPoolExecutor,
+    as_completed,
+)
+from glob import glob
+from warnings import warn
+
+import numpy as np
+import xarray as xr
+from netCDF4 import Dataset
+from rex import init_logger
+from scipy.interpolate import interp1d
+from scipy.optimize import curve_fit
+
+from sup3r.utilities.interpolation import Interpolator
+
+init_logger(__name__, log_level='DEBUG')
+init_logger('sup3r', log_level='DEBUG')
+
+
+logger = logging.getLogger(__name__)
+
+
+class LogLinInterpolator:
```

Comment:
lots going on here any my review won't be thorough. Of all the new code, it seems like this should have tests though.